applications/serial_lte_modem/prj.conf

```diff
@@ -82,6 +82,7 @@ CONFIG_SETTINGS_NVS=y
 CONFIG_NVS=y
 
 # nRF Cloud
+CONFIG_NRF_CLOUD=y
```

Comment:
There are dozen(s) of other samples/overlays where this should be added. CI run indicates this also with 100+ failures.

Reply:
This is unavoidable if it defaults as n, but then the opposite changes would be needed if it defaults as y.